grails-app/views/job/show.gsp

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+        <link rel="stylesheet" href="${resource(dir: 'css', file: 'imos.css')}" type="text/css">
+    </head>
+    <body>
+        <div class="imosHeader">
+            <div class="container">
+                <a class="btn" role="button" href="https://imos.aodn.org.au/imos123/home?uuid=8af21108-c535-43bf-8dab-c1f45a26088c">
+                    <img src="http://static.emii.org.au/images/logo/IMOS-Ocean-Portal-logo.png" alt="IMOS logo">
+                </a>
+            </div>
+        </div>
+        <div class="container">
+            <dl>
+                <dt><g:message code="job.id.label" default="Id"/></dt>
+                <dd>${job.uuid}</dd>
```

Comment:
No its a good idea - Just don't know how to do that yet.  Hopefully its easy enough.